### ISSUE TITLE
Add to osm.org edit menu

### ISSUE THREAD

**Issue by tordans** [bryceco/GoMap#758]
There is a process running to create a policy on which editors are allowed in the Edit dropdown on osm.org.
The latest update hints are about mobile editors https://github.com/openstreetmap/operations/issues/877#issuecomment-2115927350

I think that would be a good time to plan how that would work technically, which I suggest we do in this ticket.

**Comment by tordans** [comment]
My take: I wonder if the thinking in the policy is very android specific and does not fit well with the way iOS handles link-to-apps.

It says "add a generic mobile editor entry (to the menu)" and "A generic mobile editor entry would require mobile devs to come to an agreement on how this is best done".

That would be something like "mobileditor.osm.org/way/123" being linked from the menu and my iOS app would intercept this link and open it in the app. But can multiple iOS apps subscribe to the same URL? And how would I choose which app to use?

I wonder, if this is an Android specific UX that is proposed, where I remember seeing a dialogue with a choice of apps and being able to pick a default choice…

My understanding of the iOS solution would be what is described in https://github.com/bryceco/GoMap/issues/98#issuecomment-663813960 which means from osm.org edit menu the link is `https://gomaposm.com/edit?center=47.679056,-122.212559&zoom=21&view=aerial%2Beditor` which then either opens the position in GoMap or the app store to download the app. (Right?)

If this is the case, I think we should circle back to make sure the policy does not describe something that is technically not possible.

**Comment by bryceco** [comment]
Yes you are correct. It will be necessary to have individual entries for different mobile editors just as we do for iD/Josm.

**Comment by bryceco** [comment]
I see you're doing some work on the website side. Let me know if you need anything from me. Presumably the desktop editor entries have some standard URL scheme for opening them with placeholders for the current x/y/z, and maybe also for an OSM object ID. We should be able to do the exact same thing for mobile editors, right?
